plugins/inputs/dropwizard/dropwizard.go

```diff
@@ -342,4 +350,14 @@ func (*Dropwizard) DecodeJSONMetrics(r io.Reader) (metrics, error) {
 		return decodedMetrics, err
 	}
 	return decodedMetrics, nil
+}
+
+func (d *Dropwizard) FormatFloat(f float64) float64 {
+	if d.FloatFieldFormat != "" {
```

Comment:
Golang practice tends toward early exits over nested ifs:

https://talks.golang.org/2013/bestpractices.slide#4

Reply:
Good point. I blame Java. I will fix it.